server/index.js

```diff
@@ -1,9 +1,32 @@
 const express = require('express');
 const path = require('path');
+const bodyparser = require('body-parser');
 const db = require('../data/index');
 
```

Comment:
@Pfdiamond13 @fradleybrancis Consider always adding logging middleware (e.g., morgan) right away when writing a server. It makes debugging easier.